apiserver/apiserver.go

```diff
@@ -195,13 +196,19 @@ func newServer(s *state.State, lis net.Listener, cfg ServerConfig) (_ *Server, e
 	if err := srv.updateCertificate(cfg.Cert, cfg.Key); err != nil {
 		return nil, errors.Annotatef(err, "cannot set initial certificate")
 	}
-	logPath := filepath.Join(srv.logDir, "logsink.log")
-	logSinkWriter, err := newLogSinkWriter(logPath)
+
+	logSinkWriter, err := newLogSinkWriter(filepath.Join(srv.logDir, "logsink.log"))
 	if err != nil {
 		return nil, errors.Annotate(err, "creating logsink writer")
 	}
 	srv.logSinkWriter = logSinkWriter
 
+	migratedLogWriter, err := newLogSinkWriter(filepath.Join(srv.logDir, "migrated.log"))
```

Comment:
Hmm... I'm wondering about this.
Don't you think it would make more sense to have a per-model migration log file?
That way they could be deleted individually at a later date.
This would require the file to be defined at connection time rather than endpoint creation time.